### ISSUE TITLE
Faster generation?

### ISSUE THREAD

**Issue by Mixomo** [pinokiofactory/cogstudio#16]
Is it possible to use all the VRAM of the GPU (I have 24 gb) to accelerate the generation? Although I check the option to disable CPU offload, it only uses 2 or 3 GB of VRAM.

Thanks in advance.

**Comment by shutarojp** [comment]
I have same problem with my 3090. how to fix it? thanks

**Comment by Deadstarrr** [comment]
Yes. I have a 4090. I want to use all the ram available to speed up processing. It would be good if there was a VRAM limit option in the UI.

**Comment by skepsismusic** [comment]
Yes same issue, alot of unused vram and cpu cores

**Comment by Persite007** [comment]
Happy with this script. It's the way, do it yourself at home :)
Same issue. Only 2-6 Go Vram used on my RTX 3060 12 Go.
For img2vid I got good results with default settings (50 steps / 6 guid) but 1:15 to render.  :(
Easy install with Pinokio. But mentioned that version is optimized for 8Go Vram cards.
So, there are parameters somewhere...

**Comment by Deadstarrr** [comment]
Any updates on this issue?

**Comment by shutarojp** [comment]
> Any updates on this issue?

I dont think so

**Comment by Deadstarrr** [comment]
Anyone know of a separate fork that uses all of my GPU power? The pinokio install is fast and easy, but it's really slow at this point since it's not using all my vram.

**Comment by Seedmanc** [comment]
This is ridiculous, why does it only use 3-6 Gbs for you but the 8 Gbs of my 2070s is not enough?
